catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -349,18 +363,15 @@ private String getWarningMessage(
         break;
       }
     }
-    return warningMsg;
+    return warningMsg + "\\n";
   }
 
-  private String mapAliasAttributes(
+  private List<String> mapAliasAttributes(
```

Comment:
✏️ maybe we should rename this, it seems like the main purpose of this is to get the required attributes.. maybe `getRequiredAttributesAndAliases` or even just `getRequiredAttributes`